release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -370,6 +370,36 @@ This release adds improvements related to the following components and concepts.
+
+For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#absent-default-storage-class[Absent default storage class].
+
+[id="ocp-4-15-storage-lso-erase-lv-option"]
+==== Local Storage Operator option to facilitate removing existing data on local volumes is generally available
+This feature provides an optional field, `forceWipeDevicesAndDestroyAllData` defining whether or not to call `wipefs`, which removes partition table signatures (magic strings) making the disk ready to use for Local Storage Operator (LSO) provisioning. No other data besides signatures is erased. This feature is now generally available. Note that this feature does not apply to `LocalVolumeSet` (LVS).
+
+For more information, see xref:..//storage/persistent_storage/persistent_storage_local/persistent-storage-local.adoc#local-volume-cr_persistent-storage-local[Provisioning local volumes by using the Local Storage Operator].
+
+[id="ocp-4-15-storage-non-graceful-node-shutdown"]
+==== Detach CSI volumes after non-graceful node shutdown is generally available
+Starting with {product-title} 4.13, Container Storage Interface (CSI) drivers can automatically detach volumes when a node goes down non-gracefully as a Technology Preview feature. When a non-graceful node shutdown occurs, you can then manually add an out-of-service taint on the node to allow volumes to automatically detach from the node. This feature is now generally available.
+
+For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-vol-detach-non-graceful-shutdown.adoc[Detach CSI volumes after non-graceful node shutdown].
+
+[id="ocp-4-15-storage-gcp-filestore-shared-vpc"]
+==== Shared VPC is supported for the GCP Filestore CSI Driver Operator as generally available
+Shared virtual private cloud (VPC) for the Google Compute Platform (GCP) Container Storage Interface (CSI) Driver Operator is now supported as a generally available feature. Shared VPC simplifies network management, allows consistent network policies, and provides a centralized view of network resources.
+
+For more information, see xref:..//storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc#persistent-storage-csi-google-cloud-file-create-sc_persistent-storage-csi-google-cloud-file[Creating a storage class for GCP Filestore Storage].
+
+[id="ocp-4-15-storage-ibm-vpc-byok"]
+==== User-Managed encryption supports IBM VPC Block storage as generally available
+The user-managed encryption feature allows you to provide keys during installation that encrypt {product-title} node root volumes, and enables all managed storage classes to use these keys to encrypt provisioned storage volumes. The encryption feature was introduced in {product-title} 4.13, and is now supported on IBM Virtual Private Cloud (VPC) Block storage.
```

Comment:
`The encryption feature was introduced in {product-title} 4.13, and is now supported on IBM Virtual Private Cloud (VPC) Block storage.`
This is misleading, it's not an encryption feature per se, it's the ability to reuse the key used during installation. I think we should just say "The feature was introduced in {product-title} 4.13". Or if you want to be more precise

The feature was introduced in {product-title} 4.13 for GCP, Azure, AWS and is now supported on IBM Virtual Private Cloud (VPC) Block storage.

Reply:
Fixed.